CMakeLists.txt

```diff
@@ -146,44 +146,40 @@ if (XCODE)
   # Disable headermaps.
   set(CMAKE_XCODE_ATTRIBUTE_USE_HEADERMAP "NO") 
 elseif(MSVC)
-  # Disable unknown pragma warnings (e.g. for #pragma mark).
-  add_compile_options(-wd4068)
-
-  # TODO: these warnings come from llvmSupport. Since we don't want to diverge from llvmSupport by
-  # addressing these warnings , disable these for now to clean the build log.
-  # If/when we move to use LLVM's own llvmSupport, we should reenable these warnings.
-  # Unfortunately, the headers are intermingled into the public headers
-  # directory so we cannot mark the include search path as a SYSTEM directory
-  # and avoid the warnings.
   add_compile_options(
-    -wd4141 # 'inline' used more than once
-    -wd4146 # Unary minus applied to unsigned type
-    -wd4244 # Signed conversion
-    -wd4267 # Possible loss of data conversions
-    -wd4291 # operator new with no matching delete found
-    -wd4800 # Forcing value to bool 'true' or 'false'
-    -wd4996 # POSIX name for this item is deprecated
-  )
+    # Disable unknown pragma warnings (e.g. for #pragma mark).
+    "-wd4068"
+
+    # TODO: these warnings come from llvmSupport. Since we don't want to diverge from llvmSupport by
+    # addressing these warnings , disable these for now to clean the build log.
+    # If/when we move to use LLVM's own llvmSupport, we should reenable these warnings.
+    "-wd4141" # 'inline' used more than once.
+    "-wd4146" # Unary minus applied to unsigned type.
+    "-wd4244" # Signed conversion.
+    "-wd4267" # Possible loss of data conversions.
+    "-wd4291" # Operator new with no matching delete found.
+    "-wd4800" # Forcing value to bool 'true' or 'false'.
+    "-wd4996" # POSIX name for this item is deprecated.
+    "/EHsc")
 else ()
-  # Compile with C++14, without RTTI or exceptions.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -fno-exceptions")
```

Comment:
These two flags were left out of the list below, oversight? I ask because this change adds EH functions that breaks linking the Swift package manager on Android. I can always add these flags back specifically for Android, but I wonder if you really wanted to keep them here too.